Text/HTML/Parser.hs

```diff
@@ -14,6 +15,12 @@ module Text.HTML.Parser
     , Token(..)
     , TagName, AttrName, AttrValue
     , Attr(..)
+      -- * Rendering, text canonicalization
```

Comment:
I wonder whether we should split things up a bit more. This would involve moving the types (and perhaps canonicalization) to a `.Types` module, moving the parser to a `.Parser` module and the rendering to a `.Render`module.

However, what you have done here is fine for now.